application/src/main/java/org/thingsboard/server/service/security/auth/jwt/settings/DefaultJwtSettingsService.java

```diff
@@ -43,6 +46,7 @@ public class DefaultJwtSettingsService implements JwtSettingsService {
     private final AdminSettingsService adminSettingsService;
     @Lazy
     private final Optional<TbClusterService> tbClusterService;
+    private final NotificationCenter notificationCenter;
```

Comment:
The `JwtSettingsService` may run during install/upgrade phase. If notification center is not enabled under `install` profile, you should use the `Optional<notificationCenter>` similar to the 'TbClusterService'